Utilities.lua

```diff
@@ -79,13 +90,15 @@ function addon:InGroupWith(guid)
         return true, nil
     elseif strsplit("-", guid) == 'Pet' then
         return self:IsPetOwnerInRaid(guid)
-    elseif IsInRaid() then
-        for i = 1, GetNumGroupMembers() do
-            if guid == UnitGUID("Raid" .. i) then return true, nil end
-        end
     elseif IsInGroup() then
-        for i = 1, GetNumGroupMembers() do
-            if guid == UnitGUID("Party" .. i) then return true, nil end
+        if not IsInRaid() then
```

Comment:
I believe this can be simplified, hadn't fully tested but the undocumented `IsGUIDInGroup` appears to work as named.

https://github.com/valkyrnstudios/RankSentinel/compare/main...optimize_cleu#diff-1049fb2e7d8dd4a31d8ec62ab31f3d383032fc6bdb029d0b1d66c29189ee4832